lib/blender/drivers/scp.rb

```diff
@@ -77,7 +78,8 @@ class ScpDownload < Blender::Driver::Scp
       def run_command(command, session)
         begin
         rescue StandardError => e
-          ExecOutput.new(-1, stdout, e.message + e.backtrace.join("\n"))
+          stdout.rewind
```

Comment:
driver instances are shard across jobs. we may haveto wrap this in mutex. also check how this react in multi threaded driver/jobs